public/app/features/alerting/components/AlertingQueryPreview.tsx

```diff
@@ -26,12 +28,11 @@ interface Props {
   onRunQueries: () => void;
 }
 
-export const AlertingQueryPreview: FC<Props> = ({ getInstances, onRunQueries, onTest, queryRunner, queries }) => {
+export const AlertingQueryPreview: FC<Props> = ({ getInstances, onRunQueries, onTest, queries, queryRunner }) => {
   const [activeTab, setActiveTab] = useState<string>(Tabs.Query);
-  const styles = useStyles(getStyles);
-
+  const styles = getStyles(config.theme);
   const observable = useMemo(() => queryRunner.getData({ withFieldConfig: true, withTransforms: true }), []);
-  const data = useObservable<PanelData>(observable);
+  const data = useObservable(observable);
```

Comment:
Why remove the Types? That helps Typescript to infer what the observable is pushing or maybe useObservable handles this automatically?